core/tests/protractor/learnerDashboard.js

```diff
@@ -129,17 +133,17 @@ describe('Learner dashboard functionality', function() {
     // 'In Progress' section.
     player.submitAnswer('Continue', null);
     browser.ignoreSynchronization = true;
-    browser.get(general.LEARNER_DASHBOARD_URL);
+    learnerDashboardPage.get();
     general.acceptAlert();
     browser.waitForAngular();
     general.waitForSystem();
     browser.ignoreSynchronization = false;
-    element(by.css('.protractor-test-incomplete-collection-section')).click();
+    learnerDashboardPage.navigateToIncompleteCollectionSection();
```

Comment:
Let's make this ...IncompleteCollectionsSection (note the plural), since the section contains incomplete collections

Reply:
done